docs/concepts/components.mdx

```diff
@@ -9,7 +9,7 @@ type: 'docs'
 
 ## Overview
 
-**Components** are portable, interoperable WebAssembly binaries (`.wasm` files) that implement stateless logic. In wasmCloud, components typically enact the core logic of an application (for example, the API for a web application), while leaving longer-lived, reusable processes such as key-value storage to [**providers**](/docs/concepts/providers/).
+**Components** are portable, interoperable WebAssembly binaries (`.wasm` files) that implement stateless logic. In wasmCloud, components typically enact the core logic of an application (for example, the API for a web application), while leaving longer-lived, reusable [**capabilities**](/docs/concepts/capabilities/) such as key-value storage to [**providers**](/docs/concepts/providers/).
```

Comment:
This is great -- thsi sentence really makes the split between capabilities and providers clear!

Do we want to call them "capability providers" here instead of just "providers"?

Also what do you think about "abstracted, reusable capabilities"? I'm wondering if "longer lived" evokes a new direction (concerning how long something is running)